src/store/Store.ts

```diff
@@ -107,12 +107,22 @@ export interface GrafanaInfo {
   varWorkload: string;
 }
 
+export interface ServerConfig {
+  istioNamespace: string;
+  istioLabels: { [key: string]: string };
```

Comment:
Just a suggestion here, because I was changing it in another PR but I'm going to revert my changes in order to not conflict with yours: could you change the istioLabels type to this:

```
  istioLabels: { [key in IstioLabelKey]: string };
```

And define IstioLabelKey above:
```
type IstioLabelKey = 'appLabelName' | 'versionLabelName';
```

By the way, in some places I can see these labels with first letter upper case or not. What's actually given by the server, low case or upper case? It should be low case (camel case), but do we have to change on the server side as well?

Reply:
The JSON from the looks like this:

```
{
  "istioNamespace": "istio-system",
  "istioLabels": {
    "AppLabelName": "app",
    "VersionLabelName": "version"
  },
  "prometheus": {
    "globalScrapeInterval": 15,
    "storageTsdbRetention": 21600
  }
}
```

I don't know why AppLabelName and VersionLabelName are uppercase, it's been that way for a while now.  It could be changed for consistency.

Again, I will not make changes until I know whether you intend to submit a PR against this PR.

Reply:
Not sure which one will be merged first, I'm changing the backend ( https://github.com/kiali/kiali/pull/839 ) and the frontend ( https://github.com/kiali/kiali-ui/pull/997 ) ; if mine goes first, I'll submit a PR on top of yours for merging.

Reply:
@jotak OK, I have added the suggested typing.  But the keys are currently upper-cased because that is what is coming from the server.  I think we'll need a follow-up server and UI PR to convert to lowercase.

Reply:
Thanks.
FYI I'm changing it here #997 which comes with small server-side change https://github.com/kiali/kiali/pull/839 , if you want to take a look.